service/migration_manager.hh

```diff
@@ -74,6 +74,9 @@ private:
     serialized_action _schema_push;
     table_schema_version _schema_version_to_publish;
 
+    // true => schema is synchronized through Raft
```

Comment:
Please copy the cases fhen it should be true from the commit comment to here.

Reply:
Perhaps you should 1) invert the name, and 2) keep the default disabled. enable_schema_pulls = false.
This makes the name more clear and puts safety first.

Reply:
I added the comment and inverted the name.

But I kept the default enabled (`_enable_schema_pulls = true`). I find it easier to reason about that way: the boolean can only ever go from `true` to `false` during the process' lifetime; if it changes to `false` it never goes back.

If we started at `false`, we would have to set it to `true` under some cases during node start and then later at some point it could again be set to `false`.

Reply:
> But I kept the default enabled
Agreed.

Reply:
OK. My rationale for setting it to false initially since it's a safer setting which would allow us to spot bugs more if it is not managed properly. But I don't think it's worth bike shedding.